packages/bot-skeleton/src/scratch/blocks/Advanced/Functions/procedures_defnoreturn.js

```diff
@@ -90,7 +90,7 @@ Blockly.Blocks.procedures_defnoreturn = {
                     block.setFieldValue(event.newValue, 'NAME');
                 });
             }
-            removeExtraInput(this);
+            modifyProcedureBlockOnCollapse(this);
```

Comment:
```suggestion
            appendCollapseFields(this);
```

@rupato-deriv can we use this name instead?